<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorDialog</class>
 <widget class="QDialog" name="MonitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi-Monitor Configuration</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxMonitors">
     <property name="title">
      <string>Available Monitors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="monitorList">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLayout">
     <property name="title">
      <string>Monitor Layout</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="layoutPlaceholder">
        <property name="text">
         <string>Monitor layout preview will be displayed here</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f0f0f0; border: 1px solid #ccc;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="detectMonitorsButton">
       <property name="text">
        <string>Detect Monitors</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyLayoutButton">
       <property name="text">
        <string>Apply Layout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetLayoutButton">
       <property name="text">
        <string>Reset Layout</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Ready</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #666; font-style: italic;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 